Enhancements

cstoobes opened the pull request:
Couple of enhancements:
1. Version 4 of Normalize removed the table border styles because they said it was too opinionated. I think it makes sense to be in spaceBase, so I moved it to `_shared.scss` only if normalize is chosen. It still exists in the CSS reset.
2. Something I often need is a link styling reset for large clickable areas such as cards and tiles. Picture a image with text and the whole thing is inside a link, but you don't want the text to take on the default link styles and colors. If you add this helper class to the anchor tag, the color will stay dark and not underline. I pulled this from another client, so am totally open to changing the name of it. This was the original, of which I removed a lot of the resets. Do you think we should include all this? Have ideas for names?

```
.anchor--plain {
  color: inherit;
  letter-spacing: normal;
  font-family: $font-body;
  font-style: normal;
  text-transform: none;
  &:hover, &:focus {
    text-decoration: none;
  }
}
```
1. Default link hover state should be the same as focus state.
2. In newer versions of IE and Edge, the grid transition occurs on each page load, making some weird looking sliding animations. This transition doesn't have much benefit anyway, so let's remove it. This does, however, bring up the question of any transitions occurring on page load (IE seems to be the worst offender). Does anyone have a solution for that, such as [this](https://css-tricks.com/transitions-only-after-page-load/)? 

Does anyone have any other helpers they'd like to add or discuss?

athaeryn commented:
👍

cstoobes commented:
Late addition! Added .landmarklet to `_island.scss`.  

Do we want to do more with island? I know I usually remove the bottom margin from the entire group (line 11) because then I have more control over when it's there or not. That's what landmark is for, right? Thoughts?

athaeryn commented:
I agree, I don't think `.island` and `.islet` should have the bottom margin. I'm all for removing it.

99wesley99 commented:
So Carrie, this anchor reset is because when you have a link around an
element that it changes the font color, text decoration, etc? If so, I
agree and think this is an appropriate name and helper to have.

On Aug 22, 2016 10:53 AM, "Mike Anderson" <notifications@github.com> wrote:

> I agree, I don't think .island and .islet should have the bottom margin.
> I'm all for removing it.
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/space150/spaceBase/pull/61#issuecomment-241458579>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AC1dyLHbInx-v3pjrgq-fmOS2eqCGLfqks5qicX0gaJpZM4Jn2P8>
> .
>

cstoobes commented:
UPDATES:

- Removed the margin-bottom from island and islets like we discussed
- Updated the placeholder mixin prefixes for newer browser support and moved the opacity reset because it's needed in more than just Firefox
- Added a form select reset for IE10 carets

cstoobes commented:
What are people's thoughts on the `_media.scss` file?  Does anyone use it?
